generator/src/main/java/com/scottlogic/deg/generator/inputs/validation/reporters/CmdProfileValidationReporter.java

```diff
@@ -0,0 +1,33 @@
+package com.scottlogic.deg.generator.inputs.validation.reporters;
+
+import com.scottlogic.deg.generator.inputs.validation.Criticality;
+import com.scottlogic.deg.generator.inputs.validation.ValidationAlert;
+
+import java.util.List;
+
+public class CmdProfileValidationReporter implements  ProfileValidationReporter {
+
+    @Override
+    public void output(List<ValidationAlert> alerts) {
+        if (alerts.size() > 0) {
```

Comment:
Think about your nesting in this method, there are 3-4 levels deep of nesting, can you improve this.

Reply:
Either by way of calling methods for each operation, inverting `if` statements, etc.

Reply:
simplified.